PerfectDay/Table_schedule.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace PerfectDay
+{
+    public class Schedule
```

Comment:
Please use annotations for table data (https://www.entityframeworktutorial.net/code-first/dataannotation-in-code-first.aspx) (PrimarKey, MaxLength...)